operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -238,8 +238,9 @@ spec:
       version: v1
   description: "\nThe IBM Spectrum Scale CSI Operator runs within a Kubernetes cluster
     providing a means to \ndeploy and manage the CSI plugin for spectrum scale. For
-    more in depth documentation please refer\nto the [README](https://github.com/IBM/ibm-spectrum-scale-csi-operator/blob/v1.1.0/README.md).\n\nThis
-    operator should be used to deploy the CSI plugin.\n\nThe configuration process
+    more in depth documentation please refer\nto the [github repository](https://github.com/IBM/ibm-spectrum-scale-csi) 
+    which has links to official documentation.\n\nThis operator should be used to deploy 
+    the CSI plugin.\n\nThe configuration process
```

Comment:
We should just point to the KC, why github?

Reply:
I'm pointing to the github,  because  if the KC link changes we can change it easily  through an internal PR. I  want to minimize versioned  links in the CSV, because the lead time is far longer for baked changes.

With this change, we  direct the user  to the github and the `README.md` will have  all of  the support statements and documentation links. This way if we have a change in verbiage of the support statement, we can apply it without getting a third party involved.

Reply:
I'm pointing to the github,  because  if the KC link changes we can change it easily  through an internal PR. I  want to minimize versioned  links in the CSV, because the lead time is far longer for baked changes.

With this change, we  direct the user  to the github and the `README.md` will have  all of  the support statements and documentation links. This way if we have a change in verbiage of the support statement, we can apply it without getting a third party involved.